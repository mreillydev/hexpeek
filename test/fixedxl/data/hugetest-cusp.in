# A repeated write near end of file
 fffffffc,40 i @fffffff1:-0
# Kills around 4GiB limit
 ffffffff    k
100000000    k
100000001    k
# Inserts around 4GiB limit
100000001    i AA
100000000    i BB
 ffffffff    i CC
# Print
 fff00000:-0
# Create a hole
200000000,20000 i FF020305070B0D11
# Print
1fff00000:-0
