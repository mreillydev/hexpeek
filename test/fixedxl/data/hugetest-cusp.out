00000000fff00000: 11223344 11223344 11223344 11223344
*
00000000fffffff0: 11223344 11223344 11223344 223344cc
0000000100000000: 22bb44aa 22334411 22332233 44112233
0000000100000010: 44112233 44112233 22334411 22334411
0000000100000020: 22334411 22332233 44112233 44112233
0000000100000030: 44112233 22334411 22334411 112233
00000001fff00000: 00000000 00000000 00000000 00000000
*
0000000200000000: ff020305 070b0d11 ff020305 070b0d11
*
000000020001fff0: ff020305 070b0d11 ff020305 070b0d11
